projects/fsastorefrontlib/src/assets/translations/de/b2b.de.ts

```diff
@@ -5,3 +5,226 @@ export const b2b = {
     notAssigned: 'Nicht zugeordnet',
   },
 };
+
+export const organization = {
```

Comment:
Lets clean up this file as much as we can. I know that cost center translation chunk should stay bcs of know TableType enum issue but some of these can be deleted.
Examples: b2bapprovergroup, b2bmanagergroup and everything that is referring on budgets, approval permissions etc